Sources/MixedReality/Microsoft.Psi.MixedReality.UniversalWindows/MediaCaptureMicrophone.cs

```diff
@@ -205,18 +318,10 @@ private async Task<MediaFrameReader> CreateMediaFrameReaderAsync(MediaStreamType
                             // Buffer size is (number of samples) * (size of each sample)
                             byte[] audioDataOut = new byte[sampleCount * bytesPerSample];
```

Comment:
It looks like `audioDataOut` only allocates enough space for 1 channel of audio data per frame, is this correct? `sampleCount` is the number of samples _per channel_ in a frame of audio data. If that's the case, then the output buffer might not be large enough to hold the raw _n_-channel audio data in the case where you aren't downsampling to a single channel.

I think if you simply multiply `sampleCount` by `blockAlign`, that _should_ give you the correct size, taking into account the number of channels that will be in the output.

Reply:
Ah, good catch! Thanks for looking into this case :)

Reply:
Thanks! :)